Decred Wallet/Wallet Utils/TransactionNotification.swift

```diff
@@ -107,9 +108,12 @@ extension TransactionNotification: DcrlibwalletTxAndBlockNotificationListenerPro
     
     func onBlockAttached(_ walletID: Int, blockHeight: Int32) {
         // View Controllers requiring this update should call
-        // `try? AppDelegate.walletLoader.multiWallet.add(self, uniqueIdentifier: "\(self)")`
-        if Settings.beepNewBlocks && !WalletLoader.shared.multiWallet.isSyncing() {
-            self.playSound()
+        // `try? AppDelegate.walletLoader.multiWallet.add(self, uniqueIdentifier: "\(self)")
```

Comment:
Delete comment

Reply:
the comment is an explanation on how to use the function